frontend/src/pages/Login.js

```diff
@@ -119,6 +119,55 @@ class Login extends PageComponent {
     });
   }
 
+  // TODO: 서버 응답 에러 분기 처리
+  async submitRegisterForm(form) {
+    const email = form.email.value;
+    const nick = form.nick.value;
+    const password = form.password.value;
+    const passwordRe = form.passwordRe.value;
+    const registerToastMessageEl = document.getElementById('login-toast-message');
+    const registerToast = Toast.getOrCreateInstance('#login-toast');
+
+    if (!email || !nick || !password || !passwordRe) {
+      registerToastMessageEl.innerText = 'Please fill in all fields';
+      registerToast.show();
+      return;
+    }
+    if (Regex.email.test(email) === false) {
+      registerToastMessageEl.innerText = 'Invalid Email Address';
+      registerToast.show();
+      return;
+    }
+    if (Regex.nickname.test(nick) === false) {
+      registerToastMessageEl.innerText = 'Invalid Nickname';
+      registerToast.show();
+      return;
+    }
+    if (Regex.password.test(password) === false) {
+      registerToastMessageEl.innerText = 'Invalid Password';
+      registerToast.show();
+      return;
+    }
+    if (password !== passwordRe) {
+      registerToastMessageEl.innerText = 'Password does not match';
+      registerToast.show();
+      return;
+    }
+
+    const registerModal = Modal.getOrCreateInstance('#registerModal');
+    await Fetch.post('/register', { email, nick, password })
+      .then(() => {
+        registerToastMessageEl.innerText = 'Registration Successful';
+        registerToast.show();
+        registerModal.hide();
```

Comment:
성공적으로 요청 보내고 나면 input 필드를 모두 초기화해주어야 할 것 같아요! 현재는 모달 창을 다시 키면 입력했던 값이 남아있네요

Reply:
```suggestion
        registerModal.hide();
        form.nick.value = '';
        form.email.value = '';
        form.password.value = '';
        form.passwordRe.value = '';
```
이렇게 하면 되긴 하는데 요청 보냈을 때만 초기화해주는게 좋을지 모달 꺼질때마다 해주는게 좋을진 모르겠어요!

Reply:
일단 요청 성공 후에만 되게 현재위치에만 추가해놨습니다.